styleSheet.css

```diff
@@ -0,0 +1,50 @@
+ul {
+  list-style-type: none;
+}
+
+#navBar{
+  position: fixed;
+  width: 100%;
+  height: 5%;
+  top: 0;
+  background-color: #333;
+}
+
+#navBar>div{
+  position: absolute;
+  top: 0;
+  width: fit-content;
```

Comment:
what does fit-content do? I want to learn

Reply:
The element with fit-content will take enough size(width/height) to fit the content it has. Let say for example the element contains 4 other elements of 15px. 75px, 95px, 100px of width respectively then it width will be approx 285px = 15+75+95+100. And if I add any other element it will automatically resize adding the width of the new element. That is what I think it dies.